### PR TITLE
[Selenium] Fix unexpected fail of "CheckRestoringSplitEditorTest" selenium test

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Fix unexpected fail of "CheckRestoringSplitEditorTest" selenium test

### What issues does this PR fix or reference?
Issue: https://github.com/eclipse/che/issues/11302

### CODE REVIEW

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java [hunk 1]
```diff
@@ -154,6 +155,9 @@ public CodenvyEditor(
     String CONTEXT_MENU = "//div[@id='menu-lock-layer-id']/div[2]";
     String EDITOR_TABS_PANEL = "gwt-debug-multiSplitPanel-tabsPanel";
     String ACTIVE_LINE_NUMBER = "gwt-debug-cursorPosition";
+    String DEFINED_EDITOR_ACTIVE_LINE_TEMPLATE_XPATH =
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java]: `DEFINED_EDITOR_ACTIVE_LINE_TEMPLATE_XPATH` > `DEFINED_EDITOR_ACTIVE_LINE_XPATH_TEMPLATE`

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java [hunk 2]
```diff
@@ -669,16 +676,26 @@ public void goToCursorPositionVisible(int positionLine, int positionChar) {
     waitSpecifiedValueForLineAndChar(positionLine, positionChar);
   }
 
+  public void goToCursorPositionVisible(int editorIndex, int positionLine, int positionChar) {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java]: It duplicates `goToPosition(int editorIndex, int positionLine, int positionChar)` method.

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java [hunk 3]
```diff
@@ -1620,6 +1638,28 @@ public void waitTextElementsActiveLine(final String expectedText) {
     return getCursorPositionFromWebElement(currentActiveElement);
   }
 
+  public Pair<Integer, Integer> getCurrentCursorPositionFromVisible(int editorIndex) {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java]: `fromVisible` prefix name looks redundant here so as we are setting editor index.

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java [hunk 4]
```diff
@@ -1620,6 +1638,28 @@ public void waitTextElementsActiveLine(final String expectedText) {
...
+    int adoptedIndexOfEditor = editorIndex + 1;
+    String editorActiveLineXpath =
+        format(DEFINED_EDITOR_ACTIVE_LINE_TEMPLATE_XPATH, adoptedIndexOfEditor);
+
+    List<WebElement> positionWidget =
+        seleniumWebDriverHelper.waitPresenceOfAllElements(By.xpath(editorActiveLineXpath));
+
+    String cursorPositionText =
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java]: It's not quit clear what text does it mean: line where cursor is situated, or symbol at cursor position.

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java [hunk 5]
```diff
@@ -1620,6 +1638,28 @@ public void waitTextElementsActiveLine(final String expectedText) {
...
+        positionWidget
+            .stream()
+            .filter(element -> !element.getText().isEmpty())
+            .findFirst()
+            .get()
+            .getText();
+
+    List<String> positions = asList(cursorPositionText.split(":"));
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java]: Positions of what is it about?

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java (hunk 3 above)
Comment (dmytro-ndp) [reply selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java]: I would also add method's JavaDocs with explanation what kind of information returning object `Pair<Integer, Integer>` holds.

**Review by dmytro-ndp — APPROVED** [review]

**Comment by Ohrimenko1988** [comment]
ci-test

**Review by musienko-maxim — APPROVED** [review]

**Review by SkorikSergey — APPROVED** [review]

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/848/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/848//Selenium_20tests_20report/)
docker image: eclipseche/che-server:11322
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Pull request merged**